File: Dream.WebApp/ModelEntries/ReserveAccountModelEntry.cs
using System;
using System.Collections.Generic;

namespace Dream.WebApp.ModelEntries
{
    public class ReserveAccountModelEntry
    {
        public int? ReserveAccountId { get; set; }
        public string ReserveAccountName { get; set; }
        public int SecuritizationNodeId { get; set; }
        public string SecuritizationNodeName { get; set; }
        public double? InitialAccountBalanceInDollars { get; set; }
        public double? InitialAccountBalanceAsPercentage { get; set; }
        public DateTime FirstReserveAccountDrawOrDepositDate { get; set; }
        public int ReserveAccountDrawOrDepositFrequencyInMonths { get; set; }
        public string AvailableFundsRetrieverDescription { get; set; }
        public List<BalanceCapOrFloorEntry> BalanceCapOrFloorEntries { get; set; }
    }
}

File: Dream.WebApp/app/classes/modelEntries/reserveAccountModelEntry.ts
import { BalanceCapOrFloorEntry } from './balanceCapOrFloorEntry';

export class ReserveAccountModelEntry {
    ReserveAccountId: number;
    ReserveAccountName: string;
    SecuritizationNodeId: number;
    SecuritizationNodeName: string;
    InitialAccountBalanceInDollars: number;
    InitialAccountBalanceAsPercentage: number;
    FirstReserveAccountDrawOrDepositDate: Date;
    ReserveAccountDrawOrDepositFrequencyInMonths: number;
    AvailableFundsRetrieverDescription: string;
    BalanceCapOrFloorEntries: BalanceCapOrFloorEntry[];
}